storybook/src/pages/amsterdam/FormPage/FormPage.tsx

```diff
@@ -0,0 +1,112 @@
+  Column,
+  Field,
+  FieldSet,
+  FormFieldCharacterCounter,
+  Grid,
+  Heading,
+  Label,
+  Paragraph,
+  Radio,
+  Row,
+  Screen,
+  Select,
+  SkipLink,
+  TextArea,
+  TextInput,
+} from '@amsterdam/design-system-react'
+import { useState } from 'react'
+import { AppHeader } from '../common/AppHeader'
+
+export const FormPage = () => {
+  const [textareaLength, setTextareaLength] = useState(0)
+
+  return (
+    <>
+      <SkipLink href="#main">Direct naar inhoud</SkipLink>
+      <Screen maxWidth="wide">
+        <AppHeader />
+        <Grid paddingTop="small" paddingBottom="medium">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
+            <Breadcrumb>
+              <Breadcrumb.Link>Home</Breadcrumb.Link>
+            </Breadcrumb>
+            <form className="ams-gap--md" id="main" onSubmit={(e) => e.preventDefault()}>
+              <Heading>Contact</Heading>
+              <Field>
+                <Label htmlFor="body">Wat wilt u aan de gemeente vragen?</Label>
+                <Paragraph id="bodyDescription" size="small">
+                  Een duidelijke beschrijving van uw vraag helpt ons bij het behandelen.
+                </Paragraph>
+                <TextArea
+                  aria-describedby="bodyDescription"
+                  id="body"
+                  onChange={(e) => setTextareaLength(e.target.value.length)}
+                  rows={4}
+                />
+                <FormFieldCharacterCounter length={textareaLength} maxLength={1000} />
+              </Field>
+              <FieldSet aria-describedby="contactDetailsDescription" legend="Wat zijn uw gegevens?">
+                <Column gap="small">
+                  <Paragraph id="contactDetailsDescription">
+                    Wij hebben uw gegevens nodig om contact met u te kunnen opnemen.
+                  </Paragraph>
+                  <Field>
+                    <Label htmlFor="initials">Voorletters</Label>
+                    <TextInput id="initials" name="initials" />
+                  </Field>
+                  <Field>
+                    <Label htmlFor="familyName">Achternaam</Label>
+                    <TextInput autoComplete="family-name" id="familyName" name="familyName" />
+                  </Field>
+                  <Field>
+                    <Label htmlFor="city">Woonplaats</Label>
+                    <Radio name="city" value="amsterdam">
+                      Amsterdam
+                    </Radio>
+                    <Radio name="city" value="weesp">
+                      Weesp
+                    </Radio>
+                    <Radio name="city" value="anders">
+                      Anders
+                    </Radio>
+                  </Field>
+                  <Field>
+                    <Label htmlFor="email">E-mail</Label>
+                    <TextInput autoComplete="email" id="email" name="email" />
+                  </Field>
+                  <Row>
```

Comment:
The alignment between the 2 fields is off on smaller screens

Reply:
I’ve added a `wrap` prop to the `Row` component for this: #1319.